app/src/androidTest/java/net/skyscanner/backpack/compose/button/BpkButtonTest.kt

```diff
@@ -44,6 +45,7 @@ class BpkButtonTest(flavour: Flavor) : BpkSnapshotTest() {
   private val type: BpkButtonType = flavour.first
   private val size: BpkButtonSize = flavour.second
   private val icon = BpkIcon.LongArrowRight
+  private val iconDrawableRes = R.drawable.button_sample_icon
```

Comment:
can you use a different sample icon please? this might cause copyright issues.

Reply:
changed to backpack's icon